packages/rocketchat-lib/settings/server/startup.coffee

```diff
@@ -27,6 +27,9 @@ Meteor.startup ->
 	RocketChat.settings.add 'Accounts_Twitter_secret', '', { type: 'string', group: 'Accounts' }
 	RocketChat.settings.add 'Accounts_ManuallyApproveNewUsers', false, { type: 'boolean', group: 'Accounts' }
 
+	RocketChat.settings.addGroup 'General'
+	RocketChat.settings.add 'Site_Name', '', { type: 'string', group: 'General', public: true }
```

Comment:
I thought so.  But when I tested it, it didn't work.  Going to try again.  This time purging .meteor/local